core/src/main/scala/com/microsoft/azure/synapse/ml/core/schema/SchemaConstants.scala

```diff
@@ -18,6 +18,9 @@ object SchemaConstants {
   val MMLTag                    = "mml"      // MML metadata tag
   val MLlibTag                  = "ml_attr"  // MLlib metadata tag, see org.apache.spark.ml.attribute.AttributeKeys
 
+  val TreatmentResidualColumn   = "treatment_residual"
+  val OutcomeResidualColumn     = "outcome_residual"
+
```

Comment:
is this used as a prefix or a fixed column name? If its used in findUnusedCol disregard this message